boards/x86/intel_adl/doc/up7000.rst

```diff
@@ -0,0 +1,65 @@
+:orphan:
+
+.. _up7000:
+
+UP 7000 board
+#############
+
+Overview
+********
+
+UP 7000 is the 3rd generation of credit card-sized developer board of UP Boards
```

Comment:
It's not really "credit card-sized", is it? I think I saw the term "palm sized" used somewhere in the pages.

Reply:
it is, and this text is verbatim from the product web page

![image](https://github.com/zephyrproject-rtos/zephyr/assets/128251/4c9ba675-f788-41ff-a908-0421dd450272)

Reply:
I used this definition from official site https://up-board.org/up-7000/, changed to `palm-sized`

Reply:
@kartben I was confusing with the "Up Squared Pro 7000" board, which is bigger, and which is what we recently got for testing: https://up-board.org/up-squared-pro-7000/?ADLNBoard

They're both Alder Lake-based however, but seems like the "Pro" version has some more features. @finikorg I think you might need to fix the board name.

Reply:
Changed to up_squared_pro_7000